<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="inputFileButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select input file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outputFileButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select output file</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Select a gamedata.dat as input file.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>For invisible characters select original gamedata.dat as input file, new file name as output file.
Currently sprite loading is a quick hack.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="inputFilenameLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="outputFilenameLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="invisibleTrailsButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Create invisible with trails file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="invisibleButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>420</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Create fully invisible file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inputDirectoryButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select sprite input directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outputDirectoryButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select sprite output directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="inputDirectoryLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="outputDirectoryLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="invisibleLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Invisible characters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unpackSpritesButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Unpack sprites into directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="packSpritesButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pack sprites into data file</string>
    </property>
   </widget>
   <widget class="QLabel" name="spriteLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>General sprites</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="allowOverwritingCheckBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Allow overwriting while unpacking</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSpriteLoader">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSpriteLoader"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
